tox.ini

```diff
@@ -43,7 +43,7 @@ envlist =
     {py27,py34,py35,py36}-pyramid-autopatch{17,18}-webtest
     {py27,py34,py35,py36}-requests{208,209,210,211,212,213}
     {py27,py34,py35,py36}-sqlalchemy{10,11}-psycopg2{27}
-    {py27,py34,py35,py36}-psycopg2{24,25,26,27}
+    {py27,py34,py35,py36}-psycopg2{25,26,27}
```

Comment:
an update here and we're good: https://github.com/DataDog/dd-trace-py/blob/07e6b08301fe08ca648428719b4b5481d13aeb0b/docs/index.rst#supported-versions